File: AcmeCorporationWebClient/src/app/models/ticket.ts
export class Ticket {
    public firstName: string;
    public lastName: string;
    public email: string;
    public phoneNumber: string;
    public dateOfBirth: string;
    public serialNumber: string;

    constructor(firstName: string, lastName: string, email: string, phoneNumber: string, dateOfBirth: string, serialNumber: string){
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.dateOfBirth = dateOfBirth;
        this.serialNumber = serialNumber;
    }
}

File: Entities/Model/Ticket.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Entities.Model
{
    public class Ticket
    {
        public Guid Id { get; set; }
        [StringLength(50)]
        public string FirstName { get; set; }
        [StringLength(50)]
        public string LastName { get; set; }
        [EmailAddress]
        public string Email { get; set; }
        [StringLength(11)]
        public string PhoneNumber { get; set; }
        public string DateOfBirth { get; set; }
        [Required]
        public string SerialNumber { get; set; }
    }
}
